index.html

```diff
@@ -3542,14 +3543,14 @@ <h1 id="title" class="text-xl font-bold mx-auto truncate w-full">
       <!-- Start game -->
       <canvas
         id="canvas"
-        class="block fixed left-0"
+        class="block portrait:fixed left-0"
         width="400"
         height="400"
       ></canvas>
       <!-- End game -->
 
       <!-- container for both chat text and message box -->
-      <div class="flex flex-col landscape:h-[70%] landscape:grow h-full">
+      <div class="flex flex-col landscape:grow h-full">
```

Comment:
removed `h-[70%]`, was something i added arbitrarily some time back to put some margin with the bottom of the page, but now since `<main>`'s height is set with 15px buffer bottom, this makes it aligned with the canvas